fix(web,web-react)!: use padding for Checkbox, Radio, and Toggle vertical spacings (#2781)
lmc-eu/spirit-design-system: docs/migrations/web/migration-v5.md

</details>

### Checkbox, Radio, and Toggle: Vertical Spacing Uses Padding

> Non-item Checkbox, Radio, and Toggle rows previously sets its own vertical **margin** for default row spacing.

Reviewer: ?

```suggestion
Non-item Checkbox, Radio, and Toggle rows previously set their own vertical **margin** for default row spacing.
```

`Stack` with spacing (`Stack--spacing`) resets vertical margin on direct children (`margin-block: 0`) so gap-based
spacing between items stays predictable. Margin on checkbox, radio, or toggle rows inside a spaced Stack was therefore
often removed — row spacing could disappear in stacked lists.

These rows now use vertical **padding** on the layout wrapper (`py-500`) instead. Padding is not reset by Stack, so
spacing stays consistent inside and outside stacked layouts.

#### Migration Guide

Review your Checkbox, Radio, and Toggle usages and remove any `my-500` on row wrappers — use `py-500` instead. When
stacking multiple rows, use `Stack` **without** `Stack--spacing` — the built-in padding already provides row spacing;

> combining `Stack--spacing` with these rows adds redundant gap on top of padding.

Reviewer: > When stacking multiple rows, use `Stack` **without** `Stack--spacing` — the built-in padding already provides row spacing

In the FieldGroup component (which we use, for example, in the popover for the Picker), extra spacing may be desired. I would mention it here (probably only in web package, because FieldGroup in the React has spacing between children by default).

Reviewer: Applies for the Radio and Toggle as well
